src/settings/OrderApprovals/OrderApprovals.css

```diff
@@ -0,0 +1,8 @@
+.formWrapper {
+  width: 100%;
```

Comment:
I don't feel like these are required styles. Could you please elaborate more on the problem you're solving here?

Reply:
This need for stretch the block in width. without this style it looks like this
![image](https://user-images.githubusercontent.com/4868618/62921046-32760580-bdb0-11e9-9f11-2209911dfd83.png)

Reply:
I think you can achieve the same result with layout, without introducing extra specific styles 
https://ux.folio.org/storybook/?selectedKind=LayoutGrid&selectedStory=Basic%20Usage&full=0&addons=1&stories=1&panelRight=0&addonPanel=REACT_STORYBOOK%2Freadme%2Fpanel
BTW, please look at another places with settings, how it's handled there